extension/persistence/relational-jdbc/src/test/java/org/apache/polaris/extension/persistence/impl/relational/jdbc/AtomicMetastoreManagerWithJdbcBasePersistenceImplTest.java

```diff
@@ -67,4 +70,22 @@ protected PolarisTestMetaStoreManager createPolarisTestMetaStoreManager() {
             new PolarisConfigurationStore() {},
             timeSource.withZone(ZoneId.systemDefault())));
   }
+
+  private static class H2JdbcConfiguration implements RelationalJdbcConfiguration {
```

Comment:
nit: use `@PolarisImmutable` with default values.

Reply:
nevermind - there's probably not a lot of different in terms of boilerplate code :thinking: